internal/pkg/cli/task_run.go

```diff
@@ -297,11 +302,36 @@ func (o *runTaskOpts) Execute() error {
 		}
 	}
 
-	_, err := o.runTask()
+	tasks, err := o.runTask()
 	if err != nil {
 		return err
 	}
 
+	if o.follow {
+		if err := o.displayLogStream(tasks); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+func (o *runTaskOpts) displayLogStream(tasks []*task.Task) error {
+	taskDescriber := ecs.New(o.sess)
+	cwLogsGetter := cloudwatchlogs.New(o.sess)
+	logWriter := log.OutputWriter
+
+	logGroupName := fmt.Sprintf(fmtTaskLogGroupName, o.groupName)
+	ew := task.EventsWriter{
+		GroupName: logGroupName,
+		Tasks: tasks,
+	}
```

Comment:
can we define these part of the struct (initialized in a constructor) so that we can write a test case that makes `WriteEventsUntilStopped` is invoked if the `follow` flag is set